processing/src/main/java/org/apache/druid/segment/serde/ComplexColumnPartSerde.java

```diff
@@ -33,11 +34,17 @@
   @Nullable
   private final ComplexMetricSerde serde;
   private final Serializer serializer;
+  private static final Logger log = new Logger(ComplexColumnPartSerde.class);
 
   private ComplexColumnPartSerde(String typeName, Serializer serializer)
   {
     this.typeName = typeName;
     this.serde = ComplexMetrics.getSerdeForType(typeName);
+    if (this.serde == null) {
+      // Not chosing to fail here since this gets handled as
```

Comment:
nit: chosing -> choosing

Reply:
Thanks. I pushed a commit to fix the typo. Will go ahead and merge to master.